$OpenBSD: patch-i3-msg_main_c,v 1.3 2015/12/21 10:16:03 tb Exp $

pledge for i3-msg:
rpath and unix needed for talking to the i3 socket in /tmp

--- i3-msg/main.c.orig	Wed Sep 30 08:55:10 2015
+++ i3-msg/main.c	Sun Dec 20 14:42:51 2015
@@ -119,6 +119,10 @@ static yajl_callbacks reply_callbacks = {
 };
 
 int main(int argc, char *argv[]) {
+#if defined(__OpenBSD__)
+    if (pledge("stdio rpath unix", NULL) == -1)
+        err(EXIT_FAILURE, "pledge");
+#endif
     char *env_socket_path = getenv("I3SOCK");
     if (env_socket_path)
         socket_path = sstrdup(env_socket_path);
